haskell-src/lib/ChainwebData/Env.hs

```diff
@@ -226,6 +226,7 @@ data BackfillArgs = BackfillArgs
 
 data FillArgs = FillArgs
   { _fillArgs_delayMicros :: Maybe Int
+  , _fillArgs_fillErrorLog :: Maybe FilePath
```

Comment:
Do we still need this argument now that we have fixed the issue? If we're planning to keep it, we need to have a good explanation of what this log file means. I.e. which kinds of errors are logged here.